targets/frontend/src/lib/auth/token.js

```diff
@@ -56,7 +56,7 @@ async function auth(ctx) {
     }
   }
   const jwt_token = inMemoryToken;
-  if (!jwt_token) {
+  if (!jwt_token && !ctx.req) {
```

Comment:
Related?

Reply:
if no token (after trying to a renewal) and client > trying to use Router.push("/login")
there are a lot of error on client. maybe I should simplify code path by 
```
if (server) {
  try {
     renew_token
  } 
  catch() {
     response.writeHeader(302, 'Location /login")
  }
}
if (client) {
  try {
     renew_token
  } 
  catch() {
     Router.pushLogin()
  }
}

```
°

Reply:
i meant this is not ncc/rollup related :)